tests/google_tests/builtin.cpp

```diff
@@ -80,3 +80,14 @@ TEST_F(BuiltinTest, echo_n_option_multiple_arguments)
 	ASSERT_EQ(0, status_code);
 	ASSERT_STREQ("hello world !", output.c_str());
 }
+
+TEST_F(BuiltinTest, echo_n_option_multiple_arguments2)
```

Comment:
テスト名もう少しこだわりたいです。命名をちゃんと考えた上でhogehoge_2みたいな命名になってしまう場合はそのテストは要らないと思うんですよね〜。
```suggestion
TEST_F(BuiltinTest, echo_multiple_n_option)
```

Reply:
正しいと思いますm(_ _)m